calico-cloud_versioned_docs/version-3.17/release-notes/index.mdx

```diff
@@ -4,6 +4,44 @@ description: What's new, and why features provide value for upgrading.
 
 # Release notes
 
+## August 8, 2023
+
+### New and improved
```

Comment:
This looks like the release notes have been copied from Enterprise. I don't think that is what has been done before when CC has rolled out a new version of Enterprise. Did you get this from PM? I think the release notes for CC are suppose to come from PM.

Reply:
Yes, I did copy from CE RN. PM can use this as starting point and at least use working links.

Reply:
I think the releases notes in this case probably should be the same with a few minor updates